tools/Utilities/StatusCheckLogger.cs

```diff
@@ -30,7 +30,25 @@ public class StatusCheckLogger(string pathToRoot, string toolName)
     // Utility method to format the path to unix style, from the root of the repository.
     private string FormatPath(string path) => Path.GetRelativePath(pathToRoot, path).Replace(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
 
-    private void WriteMessageToConsole(string prefix, Diagnostic d) => Console.WriteLine($"{prefix}{toolName}-{d.Id}::file={FormatPath(d.file)},line={d.StartLine}::{d.Message}");
+    private void WriteMessageToConsole(string prefix, StatusCheckMessage d) => Console.WriteLine($"{prefix}{toolName}-{d.Id}::file={FormatPath(d.file)},line={d.StartLine}::{d.Message}");
+
+    /// <summary>
+    /// Log a notice from the status check to the console only
+    /// </summary>
+    /// <param name="d">The diagnostic</param>
+    /// <remarks>
+    /// Log the diagnostic information to console. These will only appear in the console window, not
+    /// as annotations on the changes in the PR. 
+    /// </remarks>
+    public void ConsoleOnlyLog(StatusCheckMessage d)
```

Comment:
I added this method because it looks like many of the status messages on the tester would be just to the console, rather than annotate the spec with info on every test.